src/twisted/web/client.py

```diff
@@ -1240,19 +1256,19 @@ def __init__(self, uri):
             self.host += ":" + str(_uri.port)
 
         self.origin_req_host = nativeString(_uri.host)
-        self.unverifiable = lambda _: False
```

Comment:
I guess that this was a bug in the previous implementation.

Reply:
Yeah, I'm not even really sure what the bug *was*. It seems like previously it would have been always unverifiable, even though it would be never-unverifiable based on runtime semantics.  Having fixed it to do what it was intending to do… no tests fail? Plus, `is_unverifiable` looks like it's never used?